.circleci/slack-notify-failure.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+
+JOB_NAME="$@"
+
+curl -X POST -H 'Content-Type:application/json' -d '{"attachments": [{"color": "#EE0000", "fallback": "Build Notification: '$CIRCLE_BUILD_URL'", "title": ":warning: homebase-garbage-collector merge failure :warning:", "text": ":x: CircleCI job `'"${JOB_NAME}"'` failed :x:\n'$CIRCLE_BUILD_URL'"}]}' $SLACK_WEBHOOK
```

Comment:
Noticed the phrasing is different in success and failure, one says `homebase-garbage-collector` and the other `Homebase Garbage Collector`. Is this intentional? Wouldn't it be nicer to have the same phrasing?